lib/CMakeLists.txt

```diff
@@ -552,6 +552,9 @@ target_include_directories(volk
     PUBLIC $<INSTALL_INTERFACE:include>
 )
 
+include_directories(${PROJECT_SOURCE_DIR}/cpu_features/include)
```

Comment:
More of a style comment, but I think it is preferable from a CMake perspective to remove this and replace it with
```
target_link_libraries(volk_obj PRIVATE cpu_features)
```
above right after the `volk_obj` target is defined. That will not only set the include directory properly for `cpu_features` but also do anything else that might be needed in the future if there are changes to `cpu_features`